pytorch_lightning/metrics/functional/classification.py

```diff
@@ -958,6 +959,89 @@ def average_precision(
     return -torch.sum((recall[1:] - recall[:-1]) * precision[:-1])
 
 
+def mean_average_precision(
```

Comment:
as it is only for object detection, it shall be reflected in name

Reply:
Do you think that perhaps that a new module should be created that is dedicated to object detection? Especially since there are some efforts being made in developing object detection specific modules in the pl_bolts repo.

Reply:
@Borda would love to get your feedback on this.